completions/_vagrant

```diff
@@ -14,6 +14,13 @@ _vagrant () {
 		fi
 		_describe -t commands "vagrant $1" vms -V1
 	}
+	getboxes() {
+        	local -a boxes
+        	boxes=(
+			$(vagrant box list | cut -d " " -f 1)
+            	)
+	        _describe -t commands "vagrant init $2" boxes -V1
```

Comment:
$2 is the second argument to the `getboxes` function. The punchline to this solution though is to remove the space and the $2 entirely and just call `getboxes` without any arguments later on.